packages/frontend/src/page/components/answer-form/answer/index.tsx

```diff
@@ -72,8 +103,8 @@ export const Answer = ({
     }
 
     return (
-        <div className="flex flex-col md:flex-row justify-between gap-3 md:gap-0 border-b-0 md:border-b dark:border-white">
```

Comment:
We can remove `md:gap-0` since we removed `gap-3`.

Reply:
Addressed in 39f8a77398538749c6c2fad147093a8f5cc1c1bd